validation-test/stdlib/Arrays.swift.gyb

```diff
@@ -231,20 +233,20 @@ ArrayTestSuite.test("COW.Slow.IndexesDontAffectUniquenessCheck") {
   
   var startIndex = a.startIndex
   var endIndex = a.endIndex
-  assert(startIndex != endIndex)
-  assert(startIndex < endIndex)
-  assert(startIndex <= endIndex)
-  assert(!(startIndex >= endIndex))
-  assert(!(startIndex > endIndex))
+  expectNotEqual(startIndex, endIndex)
+  expectLT(startIndex, endIndex)
+  expectLE(startIndex, endIndex)
+  expectGT(endIndex, startIndex)
+  expectGE(endIndex, startIndex)
```

Comment:
These checks look redundant. If `endIndex >= startIndex` then `endIndex > startIndex` and there is no need to check it. Not that we desperately need to save the CPU cycles here, but still.

Reply:
Interesting... There is `expectNotEqual(startIndex, endIndex)` at the top, given that `expectGE` is completely unnecessary, as we already know they are not equal.

Reply:
You're right. It makes sense in the String and Dictionary tests to exercise all the comparison operators, but Array's indexes are just `Int`s so these probably aren't needed.